libs/financial-aid/shared/src/lib/interfaces.ts

```diff
@@ -59,6 +59,7 @@ export interface User {
   currentApplication?: CurrentApplication
   returnUrl: ReturnUrl
   staff?: Staff
+  postalCode?: number
```

Comment:
Wondering if this should not be optional, won't we need this every time?

Reply:
Good point, isn't that future magic 🪄 ? At least for staff now we will not need it 😄  And we dont get it with our login with rafrænum skilríjum for now.